### ISSUE TITLE
solarman server change

### ISSUE THREAD

**Issue by bert1111** [sincze/Domoticz#16]
I got the solarman script working.
But an hour after I got everything in order there was a popup that said they where changing server to China.
Does the script have to be changed? 
Because I can't get it to work anymore and I'm not sure it's due to the server change or something I did.
It did work for a while

**Comment by gniersma** [comment]
Yes, noticed this also. M.ginlong is not available anymore. Redirected to the Soliscloud  https://www.soliscloud.com/#/station/stationdetail_1?id=xxxxxxxxxxxxxxx I think the api is changed
